assets/src/components/list-items/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export function ListItems( { className = '', isDisc = false, heading, items } ) {
+	const slugify = ( value ) => {
+		if ( 'string' !== typeof value ) {
+			return value;
+		}
+
+		return value
+			.toString()
+			.trim()
+			.toLowerCase()
+			.replace( /\s+/g, '-' )
+			.replace( /[^\w\-]+/g, '-' )
+			.replace( /\-\-+/g, '-' )
+			.replace( /^-+/, '-' )
+			.replace( /-+$/, '-' );
+	};
+
+	if ( isDisc ) {
+		className = classnames( 'list-items--list-style-disc', className );
+	}
+
+	return (
+		<ul className={ classnames( 'list-items', className ) }>
```

Comment:
The `classnames` util lets us add a class name conditionally.  Thanks to that, we can get rid of the conditional statement like:

```suggestion
		<ul className={ classnames( 'list-items', className, {
			'list-items--list-style-disc': isDisc,
		} ) }>
```